sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -140,6 +140,13 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val COLUMN_VECTOR_OFFHEAP_ENABLED =
+    buildConf("spark.sql.columnVector.offheap.enable")
```

Comment:
Sorry for the late comment but this should be `spark.sql.columnVector.offheap.enabled` instead of `spark.sql.columnVector.offheap.enable`?

Reply:
ah good catch! @kiszk can you send a minor PR to fix it? thanks!

Reply:
OK

Reply:
Thanks, submitted [a PR](https://github.com/apache/spark/pull/19794)